readthedocs_theme/templates/includes/topnav.html

```diff
@@ -1,5 +1,5 @@
 {% macro menu_main() %}
-  <a class="item{% if page and page.slug == 'about' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/about/">About</a>
+  <a class="item{% if page and page.slug == 'about' %} active" aria-current="page{% endif %}" href="{{ SITEURL }}/how-it-works/">How it works</a>
```

Comment:
"How it Works" doesn't look good enough for me to want to break the pattern of single word entries in the top navigation. The website has a few entries right now, but I'd like more, and the application will have more top navigation entries.